Makefile

```diff
@@ -185,6 +185,11 @@ docopen: out/doc/api/all.html
 docclean:
 	-rm -rf out/doc
 
+run-ci:
+	$(PYTHON) ./configure $(CONFIG_FLAGS)
```

Comment:
Probably not necessary but it doesn't hurt either, does it? We already invoke it like this in a couple other places in the makefile.